lib/scrape.js

```diff
@@ -84,13 +95,19 @@ async function fetchOrgsWithData() {
   const orgs = await fetchOrgs()
   const fetchingLeaders = orgs.map(org => fetchLeaders(org.id))
   const fetchingGitHub = orgs.map(org => findOrganization(org))
+  const fetchingChat = orgs.map(org => chattie(org.irc_channel))
   const orgLeaders = await Promise.all(fetchingLeaders)
   const orgGitHub = await Promise.all(fetchingGitHub)
+  const orgChats = await Promise.all(fetchingChat)
 
   return orgs.map((org, index) =>
     Object.assign(org, {
       leaders: orgLeaders[index],
       github: orgGitHub[index],
+      chat: {
+        url: orgChats[index].url,
+        image: CHAT_IMAGES[chattie.CHAT[orgChats[index].type]],
```

Comment:
I'd prefer the chat platform type to be included also, to give a bit of clarity in the JSON.